Docs/user_guide/model_quantization.rst

```diff
@@ -2,51 +2,51 @@
-without needing an actual target platform. Note that to create a simulation model, AIMET uses representative data
-samples to compute per-layer quantization encodings.
+
+This section briefly describes how AIMET's quantization features apply to typical use cases.
+
+Quantization simulation
+    AIMET enables you to simulate running models on quantized targets. This helps you estimate on-target accuracy without requiring you to move the model to a quantized target platform.
+    
+    A quantization simulation workflow is illustrated here:
 
     .. image:: ../images/quant_use_case_1.PNG
 
-2. **Post-Training Quantization (PTQ)**: PTQ techniques attempt to make a model more quantization friendly without
-requiring model re-training/fine-tuning. PTQ (as opposed to fine-tuning) is recommended as a first step in a
-quantization workflow due to the following advantages:
+Post-training quantization (PTQ)
+    PTQ techniques make a model more quantization-friendly without requiring model retraining or fine-tuning. PTQ is recommended as a first step in a quantization workflow because:
 
-- No need for the original training pipeline; an evaluation pipeline is sufficient
-- Only requires a small unlabeled dataset for calibration (can even be data-free in some scenarios)
-- Fast, simple, and easy to use
+    - PTQ does not require the original training pipeline; an evaluation pipeline is sufficient
+    - PTQ requires only a small, unlabeled dataset for calibration
+    - PTQ is fast and easy to use
+    
+    The PTQ workflow is illustrated here:
 
     .. image:: ../images/quant_use_case_3.PNG
 
-Note that with PTQ techniques, the quantized model accuracy may still have a gap relative to the floating-point model.
-In such a scenario, or to even further improve the model accuracy, fine-tuning is recommended.
+    With PTQ techniques, model accuracy may still be reduced. In such cases, fine-tuning is recommended.
 
-3. **Quantization-Aware Training (QAT)/Fine-Tuning**: QAT enables a user to fine-tune a model with quantization
-operations inserted in network graph, which in effect adapts the model parameters to be robust to quantization noise.
-While QAT requires access to a training pipeline and dataset, and takes longer to run due to needing a few epochs of
-fine-tuning, it can provide better accuracy especially at low bitwidths. A typical QAT workflow is illustrated below.
+Quantization-aware training (QAT) and fine-tuning
+    QAT enable you to fine-tune a model with quantization operations inserted in the network graph. In effect, it makes the model parameters robust to quantization noise.
+
+    Compared to PTQ, QAT requires a training pipeline and dataset and takes longer because it needs some fine-tuning, but it can provide better accuracy, especially at low bitwidths.
+    
+    A typical QAT workflow is illustrated here:
 
     .. image:: ../images/quant_use_case_2.PNG
 
-AIMET Quantization Features
+_aimet-quantization-features:
```

Comment:
Does this page even exist? `_aimet-quantization-features:` Let's make sure before cross-referencing it.

Reply:
It's a reference within the same file.